.github/workflows/ci.yml

```
@@ -0,0 +1,10 @@
+name: ci
+on: [push]
```

I think we need to add `pull_request` here for it to run on PRs as well:
```suggestion
on: [push, pull_request]
```